docs/kcl/std.json

```diff
@@ -211843,8 +211843,8 @@
   },
   {
     "name": "tangentialArc",
-    "summary": "Starting at the current sketch's origin, draw a curved line segment along",
-    "description": "some part of an imaginary circle of the specified radius.\nThe arc is constructed such that the last line segment is placed tangent to the imaginary circle of the specified radius. The resulting arc is the segment of the imaginary circle from that tangent point for 'offset' degrees along the imaginary circle.",
+    "summary": "Draw a curved line segment along some part of an imaginary circle of the specified radius.",
+    "description": "If `relative` is true, the curve starts at the end of the previous path segment (i.e. the location of the \"pen\" which draws these lines). If `relative` is false, starts from the current sketch's origin.\nThe arc is constructed such that the last line segment is placed tangent to the imaginary circle of the specified radius. The resulting arc is the segment of the imaginary circle from that tangent point for 'offset' degrees along the imaginary circle.",
```

Comment:
@paultag wdyt of my docs here?

Reply:
excellent docs; this is great, super good change.